packages/bruno-app/src/components/RequestPane/Auth/BearerAuth/index.js

```diff
@@ -11,9 +11,8 @@ const BearerAuth = ({ item, collection }) => {
   const dispatch = useDispatch();
   const { storedTheme } = useTheme();
 
-  const bearerToken = item.draft
-    ? get(item, 'draft.request.auth.bearer.token')
-    : get(item, 'request.auth.bearer.token');
+  const bearerToken =
```

Comment:
This does work, but I don't think it's a "fancy" solution. How about giving get a default value?

```suggestion
  const bearerToken = item.draft
    ? get(item, 'draft.request.auth.bearer.token', '')
    : get(item, 'request.auth.bearer.token', '');
```